docs/developers/mobile.md

```diff
@@ -0,0 +1,34 @@
+# Mobile Development
+
+## Testing the client on a mobile device
+
+If you have made changes to the client that could affect the mobile experience, it is a good idea to test them on a real device. Such changes should ideally be tested with at least current versions of iOS Safari and Chrome for Android.
+
+1. Make sure your development system and mobile device are on the same local network.
+1. Get the IP address or host name of your development system (`$HOSTNAME` in the steps below). You can do this using the `hostname` terminal command on Mac/Linux.
```

Comment:
Thank-you for the suggestion but I think this is way too verbose and distracts from the Hypothesis-specific issues. I've added a sentence noting that if the hostname does not have a `.home` or `.local` suffix, the user may need to add it, and if that doesn't work (for many reasons), they should just use the IP address.

Regarding `$HOSTNAME`, I take your point but I think setting an env var might give the misleading impression that the commands make use of this env var name internally, as opposed to it just being a placeholder in the docs. I've left it for now and will wait for feedback from people trying to actually follow these docs.